common/actions/transaction/actionCreators/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './sign';
 export * from './broadcast';
 export * from './current';
 export * from './sendEverything';
+export * from '../../../containers/Tabs/ScheduleTransaction/actions/transaction/actionCreators';
```

Comment:
Prefer non-relative exports for easier refactoring (`containers/Tabs/...` instead of `../../../`)